src/BenchmarksDriver/Program.cs

```diff
@@ -402,26 +434,71 @@ public static int Main(string[] args)
 
                 if (clientJob.State == ClientState.Completed && !string.IsNullOrWhiteSpace(sqlConnectionString))
                 {
-                    await WriteResultsToSql(
+                    // Load latest state of server job
+                    LogVerbose($"GET {serverJobUri}...");
+                    response = await _httpClient.GetAsync(serverJobUri);
+                    responseContent = await response.Content.ReadAsStringAsync();
+
+                    LogVerbose($"{(int)response.StatusCode} {response.StatusCode} {responseContent}");
+
+                    serverJob = JsonConvert.DeserializeObject<ServerJob>(responseContent);
+
+                    Log("Writing results to SQL...");
+
+                    await WriteJobsToSql(
+                        serverJob: serverJob, 
+                        clientJob: clientJob,
                         connectionString: sqlConnectionString,
                         scenario: scenario,
```

Comment:
`scenario` is also available as `serverJob.Scenario`.  I would just remove the local variable entirely and replace it with `serverJob.Scenario`.